glass/user.py

```diff
@@ -89,7 +117,11 @@ def write(self, data=None, /, **columns) -> None:  # type: ignore[no-untyped-def
 
 
 @contextmanager
-def write_catalog(filename, *, ext=None):  # type: ignore[no-untyped-def]
+def write_catalog(
+    filename: pathlib.Path,
+    *,
+    ext: str = "",
```

Comment:
Not the same as `ext=None`.

Reply:
I'm aware by what would `ext` ever be `None`? My understanding is "a.csv" + "" = "a.csv"

Reply:
Oh, right. This `ext=` is the FITS extension name. Extensions don't need to have a name, so that's why `None` (no name, use extension number) and `""` (empty string) are different

Reply:
Okay, makes sense. I couldn't find much documentation on FITS, will change